tests/test_api_monitoring.py

```diff
@@ -13,13 +14,13 @@ def test_01_get_stats(self):
         # create some statistics
 
         write_stats("key1", 1)
-        write_stats("key2", "A")
+        write_stats("key2", 50)
```

Comment:
Why would you remove the string as a stats value?
There are stats, I think, that can be strings.

Reply:
Actually `write_stats()` already annotates the value as `int`:
https://github.com/privacyidea/privacyidea/blob/315c30927382bab04bc3ea76321be7e418a2e3e5/privacyidea/lib/monitoringstats.py#L68
Also the db model uses Integers:
https://github.com/privacyidea/privacyidea/blob/cb4296718c92db9d2ab06bb45d95cb19405555e3/privacyidea/models.py#L3018
I guess SQLite isn't too picky about the data types...

Reply:
Oups. I was not aware of this. Thx.